python/cudf/cudf/pylibcudf_tests/test_string_find.py

```diff
@@ -0,0 +1,177 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+import numpy as np
+import pyarrow as pa
+import pytest
+from utils import assert_column_eq
+
+import cudf._lib.pylibcudf as plc
+from cudf._lib.scalar import DeviceScalar
+
+
+@pytest.fixture()
```

Comment:
Any reason that you dropped module scopes? Also, this fixture has trailing parentheses while the others don't. We should be stylistically consistent. I'd probably make them all module scope unless there's a reason not to, which means keeping parentheses.